src/app/support/page.js

```diff
@@ -1,5 +1,24 @@
+import Connect from "@/components/support/Connect";
+import { SOCIALS } from "@/data/support/Socials";
+
 const Page = () => {
-  return <>Support</>;
+  return (
+    <>
+      <div className="text-center text-biscuits-brown-200 font-shrikhand p-1 -translate-y-1 text-3xl">
+        {"Connect with us!"}
+        <div className="flex space-x-6 justify-center">
```

Comment:
![image](https://github.com/acm-ucr/h4b-website/assets/91030482/b3853204-25fd-48fd-ad3c-353ad8318ca9)

space them out a little more

Reply:
i think some icons like instagram are not centered, i would try using text-center or justify-center

Reply:
I think I got most of them to be center except instagram, but I've tried text an d justify center and no changes. Is there another one that works for it?

Reply:
wait it should be items-center 😭 my fault
but it should be in the Connect component